.travis.yml

```diff
@@ -32,6 +33,7 @@ script:
   - if [[ $coverage = 1 ]]; then phpdbg -qrr vendor/bin/behat --profile coverage; else vendor/bin/behat; fi
   - if [[ $coverage = 1 ]]; then phpdbg -qrr phpcov.phar merge --clover build/logs/clover.xml build/cov; fi
   - tests/Fixtures/app/console api:swagger:export > swagger.json && swagger validate swagger.json && rm swagger.json
+  - if [[ $lint = 1 ]]; then php php-cs-fixer.phar fix --dry-run --diff; fi
```

Comment:
IMO StyleCI is a separate check. It should not be a replacement.

At least with this we can be sure that our `.php_cs` config is up-to-date. This should encourage running `php-cs-fixer`. Of course we can also add a pre-commit hook, but I'm not familiar with that.